pkg/manifest/anaconda_installer_iso_tree.go

```diff
@@ -25,6 +25,15 @@ const ( // Rootfs type enum
 	ErofsRootfs                          // Create a plain erofs rootfs
 )
 
+type ISOBootType uint64
+
+// These constants are used by the ISO images to control the type of bootable iso
+const ( // ISOBoot type enum
+	UEFIOnlyISOBoot ISOBootType = iota // Only boot with grub2 UEFI
```

Comment:
Can we call this `Grub2ISOBootUefiOnly` or something, I know its a bunch longer but the other two here include the name of the bootloader used.

Reply:
Renamed it to `Grub2UEFIOnlyISOBoot` so it's consistent with the others.